lib/scripts/create_keywords_yaml.rb

```diff
@@ -0,0 +1,18 @@
+INDENT = "  "
+
+File.open("keywords.yaml", "w") do |file|
+  tags = {}
+
+  GenericFile.all.each do |gf|
+    gf.tag.each do |tag|
+      slug = tag.parameterize
+
+      if tags[slug]
+        next
+      else
+        tags[slug] = tag
+        file.write("- id: #{slug}\n#{INDENT}title:\n#{INDENT*2}en: \"#{tag}\"\n")
```

Comment:
Updated to use double quote character to wrap tag. This should handle any special characters that may appear in tags.